py-polars/polars/selectors.py

```diff
@@ -124,9 +125,7 @@ def expand_selector(
     return tuple(target.select(selector).columns)
 
 
-def _expand_selectors(
-    frame: DataFrame | LazyFrame, items: Any, *more_items: Any
-) -> list[Any]:
+def _expand_selectors(frame: DataFrame | LazyFrame, *items: Any) -> list[Any]:
```

Comment:
I updated the logic here to match the expression parsing util. We should deduplicate this logic, but this works for now. Had to make these changes to support Iterable input.

Reply:
Did some refactoring in other PRs and this now has no duplication. The function still needs a refactor though.